lib/openxml/parts/rels.rb

```diff
@@ -44,8 +48,6 @@ def to_xml
         end
       end
 
-
-
```

Comment:
We use triple line-breaks in some places to separate "paragraphs" of code in a file — FWIW — more as a subjective communication tool than with any hard-and-fast Do's and Don't's.